pkg/controller/podautoscaler/metrics/metrics_client.go

```diff
@@ -119,6 +119,11 @@ func (h *HeapsterMetricsClient) GetCpuConsumptionAndRequestInMillis(namespace st
 	if err != nil {
 		return 0, 0, time.Time{}, fmt.Errorf("failed to get pod list: %v", err)
 	}
```

Comment:
The permissions on this file changed to executable, please change the permissions back to what they were before.